dev/kyuubi-tpcds/src/main/scala/org/apache/kyuubi/tpcds/benchmark/RunBenchmark.scala

```diff
@@ -65,6 +66,9 @@ object RunBenchmark {
       opt[String]('r', "results-dir")
         .action((x, c) => c.copy(resultsDir = x))
         .text("dir to store benchmark results, e.g. hdfs://hdfs-nn:9870/pref")
+      opt[String]('w', "white-list")
```

Comment:
how about 'q' "queries" ? cc @ulysses-you

Reply:
Fine with `queries`, will upload later.

Reply:
> how about 'q' "queries" 

+1